tests/sentry/hybridcloud/test_organization.py

```diff
@@ -360,3 +360,20 @@ def test_send_sso_unlink_emails() -> None:
     assert len(mail.outbox) == 2
     assert "Action Required" in mail.outbox[0].subject
     assert "Single Sign-On" in mail.outbox[0].body
+
+
+@django_db_all(transaction=True)
+@all_silo_test
+def test_get_aggregate_project_flags() -> None:
+    org = Factories.create_organization()
+    project1 = Factories.create_project(organization_id=org.id, name="test-project-1")
+    Factories.create_project(organization_id=org.id, name="test-project-2")
+    flags = organization_service.get_aggregate_project_flags(organization_id=org.id)
+    assert flags.has_insights_http is False
+
+    with assume_test_silo_mode_of(Project):
+        project1.flags.has_insights_http = True
+        project1.update(flags=F("flags").bitor(Project.flags.has_insights_http))
```

Comment:
It would be good set a different flag on the second project you create, then you could assert that the results contains flags from both projects.